samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/data/repositories/PreferenceRepository.kt

```diff
@@ -31,10 +31,10 @@ class PreferenceRepositoryImp(private val dataStore: DataStore<Preferences>) :
         dataStore.edit { preferences ->
             preferences[CDP_API_KEY] = configuration.cdpApiKey
             preferences[SITE_ID] = configuration.siteId
-            preferences[API_KEY] = configuration.apiKey
-            configuration.trackUrl?.let { preferences[TRACK_API_URL_KEY] = it }
-            preferences[BACKGROUND_QUEUE_SECONDS_DELAY] = configuration.backgroundQueueSecondsDelay
-            preferences[BACKGROUND_QUEUE_MIN_NUM_TASKS] = configuration.backgroundQueueMinNumTasks
+            configuration.apiHost?.let { preferences[API_HOST_KEY] = it }
+            configuration.cdnHost?.let { preferences[CDN_HOST_KEY] = it }
```

Comment:
```suggestion
            preferences[API_HOST_KEY] = configuration.apiHost
```

I was expecting to see this, so that you could set the preferences value back to null if you wish. 

Is there a reason to first check for null?

Reply:
`preferences[API_HOST_KEY]` actually doesn't allow `null` values, so it has to be non-null